packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -14,6 +14,35 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * A class that manages media settings state and performs media settings operations.
```

Comment:
This seems like two very different responsibilities; why are these a single object? I would expect one configuration object, and one one (or more, but one is fine) object for AV* wrappers.

Reply:
splitted into `FLTCamMediaSettings` and `FLTCamMediaSettingsAVWrapper`. Each in own file pair.